distsql/distsql.go

```diff
@@ -35,9 +39,25 @@ func Select(ctx context.Context, sctx sessionctx.Context, kvReq *kv.Request, fie
 		hook.(func(*kv.Request))(kvReq)
 	}
 
+	child := tracing.ChildSpanFromContxt(ctx, "distsql_select")
+	defer child.Finish()
+
 	if !sctx.GetSessionVars().EnableStreaming {
 		kvReq.Streaming = false
 	}
+
+	for _, key := range kvReq.KeyRanges {
```

Comment:
We don't need to trace every key range.
In index lookup request, there can be thousands of ranges.
This has too much cost for index lookup requests.